elements/pfe-tooltip/demo/pfe-tooltip.js

```diff
@@ -0,0 +1,18 @@
+import '@patternfly/pfe-tooltip';
```

Comment:
this isn't loading on deploy preview

Reply:
if this is good by you, @brianferry go ahead and resolve

Reply:
Almost ready on this PR, looking at @popperjs/core not being updated with the new docs importmaps.